.changeset/little-steaks-rule.md

```diff
@@ -0,0 +1,70 @@
+---
+'astro': patch
+---
+
+The SVG import feature introduced behind a flag in [v5.0.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#500) is no longer experimental and is available for general use.
```

Comment:
The feature **is still experimental**, judging by the code. I think, maybe, there's been a misunderstanding on how to reword the changeset. Here's the relevant paragraph: https://contribute.docs.astro.build/docs-for-code-changes/experimental-feature-docs/#breaking-changes-while-still-experimental

We just need to add at the top of the changeset the following:

```
**BREAKING CHANGE to the experimental SVG configuration only**
```

Reply:
Oh my gosh, how embarrassing 😂 Sorry for that! I'll get that updated today

Reply:
Just checking @ematipico -- taking this out of experimental *might* be a plan for Launch week? Which now would be our next minor release. So, not sure whether this PR is intended as a patch first, or whether this will merge with the next minor at launch.

But yes, either the code needs to catch up to that, or the changeset needs to only discuss the remove the features. I'll wait to polish up the changeset until we have decided what the goal of this PR is!

Reply:
@sarah11918 that's the plan, but that's not for this PR. We still have a memory leak that we need to fix, and if we can't fix it, we can't make the feature stable.

Reply:
Do you mind tagging that issue for me here? I'm having trouble finding where you pointed it out.

Reply:
@stramel There's an issue for that, but there's a Discord thread: https://discord.com/channels/830184174198718474/1349441804193239150